examples/eks_cluster_webshop/main.tf

```diff
@@ -64,9 +64,6 @@ module "castai-eks-cluster" {
   aws_instance_profile_arn      = module.castai-eks-role-iam.instance_profile_arn
 
   subnets         = var.subnets
-  # As for now (05.05.2022) you need to fill security groups after cluster onboarding with values from plan.  
```

Comment:
How this example will work with SGs from now on?

Reply:
In that example we didn't use SGs it were added only because of this bug